### PR TITLE
docs(drop-container): prevent default browser drag and drop behavior

### LINKED ISSUE #5732: By default if using the FileUploader component(s) files can be DnD-ed everywhere
### Summary

If using the `FileUploaderDropContainer` component the default behaviour is as follows:

1. Files, e.g. text files can be dropped onto the drop area which is correct. The logic that is wired to the drop area will kick-in which is also correct.
2. If DnD-ing a text file it is by default also possible to drop it anywhere in the user agent's (browser) real estate. The user agent will then render the contents of the text file.

Instead if a SPA (web site) is making use of a `FileUploaderDropContainer` component it should be considered that in that case dropping files outside the drop area must not be possible. If doing so this would (A) prevent accidental drops and thus requiring the user to browser back cause the context of the application just vanished and (B) it also makes it more clear where files can be dropped and where not.  

### Justification

Improved usability. See above under summary.

### PR DESCRIPTION
**Pull request by emyarod**
Closes #5732

This PR updates the drag and drop file uploader examples to include a way to prevent browser default behavior (described in the ticket)

#### Changelog

**Changed**

- update storybook and code sandbox examples to override default browser drag and drop behavior
- update code sandbox example `package.json`

#### Testing / Reviewing

Drag and drop a file onto the browser outside of the drop target. The browser should not try to open or download the file

### CODE REVIEW

**Review by joshblack — APPROVED** [review]

File: packages/react/examples/drag-and-drop-file-uploader/package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "carbon-components": "latest",
     "carbon-components-react": "latest",
+    "carbon-icons": "latest",
```
Comment (asudoh) [inline packages/react/examples/drag-and-drop-file-uploader/package.json:8]: Just wanted to see if this is intentional?

**Review by asudoh — APPROVED** [review]
Thank you for your quick doc update @emyarod!

File: packages/react/examples/drag-and-drop-file-uploader/package.json (hunk 1 above)
Comment (emyarod) [reply packages/react/examples/drag-and-drop-file-uploader/package.json:8]: yes, on `master` branch the example will not load without this dependency https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/react/examples/drag-and-drop-file-uploader

Comment (asudoh) [reply packages/react/examples/drag-and-drop-file-uploader/package.json:8]: Got it - Thanks @emyarod for clarifying!

**Comment by asudoh** [comment]
@emyarod thinks this is ready to merge - Thanks @emyarod!

**Pull request merged**